public/usage-examples/graphics/has_font/has_font-1-simple.cpp

```diff
@@ -0,0 +1,24 @@
+#include "splashkit.h"
+
+int main()
+{
+    open_window("Has Font", 800, 600);
+    clear_screen();
+
+    //Check if the font exit or not
+    bool fontCheck = has_font("NORMAL_FONT");
+    // Display the text to show the result
+    
+    if(fontCheck)
+    {
+        draw_text("Font find!", COLOR_BLACK, 300, 300);
```

Comment:
On this line and on line 17, change 'find' to 'found' to keep things consistent with other code files